buffer/src/main/java/io/netty5/buffer/DefaultCompositeBuffer.java

```diff
@@ -574,6 +574,34 @@ public int transferTo(WritableByteChannel channel, int length) throws IOExceptio
         return totalBytesWritten;
     }
 
+    @Override
+    public int transferTo(FileChannel channel, long position, int length) throws IOException {
+        if (!isAccessible()) {
+            throw bufferIsClosed(this);
+        }
+        length = Math.min(readableBytes(), length);
+        if (length == 0) {
+            return 0;
+        }
+        checkReadBounds(readerOffset(), length);
+        int totalBytesWritten = 0;
+        try (var iterator = forEachComponent()) {
```

Comment:
Let's not use `var` and go with the explicit object type.

Reply:
Yeah, this is fine.